### PR TITLE
Add documentation for mozilla.org

### PR DESCRIPTION
**Pull request by peiying2**
- Add a doc for mozilla.org
- Add a README.md
- Updated the SUMMARY.md

### CODE REVIEW

**Comment by peiying2** [comment]
@flodolo I did the best I could, but there are two issues that I'm aware of but couldn't resolve:
- Line 40: `From file changes in [mozilla/bedrock@ 3061d26](http://mozilla/bedrock@%203061d26)` - couldn't hid the link.
- Line 66: Cross reference to the section within the doc (add or remove a file).

**Comment by flodolo** [comment]
Please review the fixes for the future
https://github.com/mozilla-l10n/documentation/pull/291/commits/1f5fb9e258a27caef92eef9b90ccc83320e9701b

Here are a few things that I fixed:
* Several links were wrong at the beginning.
* Reorganized the *Review PRs and address common errors* section, moved error caught by linters here.
* Don't use bold for titles, use title levels.
* A line break in the file doesn't mean a line break in the rendered file. Use `<br>`.

> Line 40: From file changes in [mozilla/bedrock@ 3061d26](http://mozilla/bedrock@%203061d26) - couldn't hid the link.

You can't have a link within a code block. I've removed it.

> Line 66: Cross reference to the section within the doc (add or remove a file).

Anchors have - instead of spaces

**Review by flodolo — APPROVED** [review]

File: src/products/mozilla_org/manage_review_mozilla_org.md [hunk 1]
```diff
@@ -0,0 +1,194 @@
...
+  * Correct: `{ -brand-name-mozilla-account }`or `{ -brand-name-mozilla-accounts }`.
+  * Incorrect: `{ -brand-name-mozilla } account` or `{ -brand-name-mozilla } accounts`.
+* *Firefox Browser*: The marketing team is phasing out the placeholder `{ -brand-name-firefox-browser }`. Instead, use the `{ -brand-name-firefox } browser` for both singular and plural forms.
+* Third-Party brands and products: Brand and product names from third parties — like Google, Apple, etc. — are no longer coded as Fluent terms in new content.
+
+#### Content issues
+
+Here are some issues that commonly occur:
+* Content changes without new string IDs: If a string is revised, a version number must be added to the ID. [This is necessary](https://mozilla-l10n.github.io/documentation/localization/making_string_changes.html#why-is-it-necessary-to-use-new-ids) because updates to strings without changes to IDs will not trigger updates in Pontoon for strings already localized.<br>
+Example: Update the string ID from ``features-fast-firefox-is-powered-by-the-world` to `features-fast-firefox-is-powered-by-the-world-v2`.
```
Comment (peiying2) [inline src/products/mozilla_org/manage_review_mozilla_org.md:60]: Update the string ID from ``features-fast...` - should have one opening `.

Comment (flodolo) [reply src/products/mozilla_org/manage_review_mozilla_org.md:60]: Please update it (it's your doc in the end 😉). No need to go through a PR if you want to skip it.

Comment (peiying2) [reply src/products/mozilla_org/manage_review_mozilla_org.md:60]: Noted. I'm going through the issues corrected in this document to see if anything needs to be called out in the style guide. Will comment in the style guide PR today.

**Pull request merged**